pkg/util/liveconfig/hive.go

```diff
@@ -116,3 +117,17 @@ func (p *prod) AdoptByHive(ctx context.Context) (bool, error) {
 	}
 	return false, nil
 }
+
+func (p *prod) OCMValidClientIDs() ([]string, error) {
+	// TODO: Replace with RP Live Service Config (KeyVault)
+	ocmValidClientIDString := os.Getenv(ocmValidClientIDs)
```

Comment:
Consider a comment here on where we get/set ocmValidClientIDs and how to add new ones

Reply:
I can add a comment on the format the IDs will need to be in in RP-Config. However, I don't know how we will be obtaining these IDs from OCM.

Reply:
Going through this we will have to make some additional changes to this branch so that we can properly pipe in configuration values in RP-Config into the environment of the RP